README.md

```diff
@@ -103,11 +103,11 @@ bg := `{{ Background "#0000ff" "Blue Background" }}`
 wrap := `{{ Bold (Underline "Hello World") }}`
 
 // parse and render
-tpl = tpl.Parse(bold)
+tpl, _ = tpl.Parse(bold)
```

Comment:
Thanks for catching this! Since this isn't mean to be directly copy/pasted, and it would improve readability, I'd actually suggest giving the second parameter the name `err`.